### PR TITLE
Fixes clowns blowing shit up not being logged 50% of the time

### PR DESCRIPTION
**Pull request by silicons**
clowny clown clown :clown_face:

### CODE REVIEW

**Comment by AnturK** [comment]
Don't copypaste, add argument/s to the standard function

**Comment by KingPhilipIII** [comment]
Why the clown in specific?

File: code/game/objects/items/grenades/grenade.dm [hunk 1]
```diff
@@ -50,26 +44,27 @@
...
 
-/obj/item/grenade/proc/preprime(mob/user)
-	if(user)
+/obj/item/grenade/proc/log(mob/user)
+	var/turf/T = get_turf(src)
+	var/area/A = get_area(T)
+	var/message = "[ADMIN_LOOKUPFLW(user)]) has primed a [name] for detonation at [ADMIN_COORDJMP(T)]"
+	GLOB.bombers += message
+	message_admins(message)
+	log_game("[key_name(user)] has primed a [name] for detonation at [A.name] [COORD(T)].")
```
Comment (Cyberboss) [inline code/game/objects/items/grenades/grenade.dm]: \a [src]

File: code/game/objects/items/grenades/grenade.dm [hunk 2]
```diff
@@ -50,26 +44,27 @@
...
-			if(iscarbon(user))
-				var/mob/living/carbon/C = user
-				C.throw_mode_on()
 
-/obj/item/grenade/proc/preprime(mob/user)
-	if(user)
+/obj/item/grenade/proc/log(mob/user)
+	var/turf/T = get_turf(src)
+	var/area/A = get_area(T)
+	var/message = "[ADMIN_LOOKUPFLW(user)]) has primed a [name] for detonation at [ADMIN_COORDJMP(T)]"
```
Comment (Cyberboss) [inline code/game/objects/items/grenades/grenade.dm]: \a [src]

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Comment by kevinz000** [comment]
fixing this soon dont yell at me <3

**Comment by Cyberboss** [comment]
REEEE

**Comment by PraiseRatvar** [comment]
WHY THE CLOWN IN SPECIFIC?

**Comment by kevinz000** [comment]
fixed

File: code/game/objects/items/grenades/grenade.dm [hunk 3]
```diff
@@ -50,26 +44,27 @@
...
 			preprime(user)
-			if(iscarbon(user))
-				var/mob/living/carbon/C = user
-				C.throw_mode_on()
 
-/obj/item/grenade/proc/preprime(mob/user)
-	if(user)
+/obj/item/grenade/proc/glog(mob/user, turf/T)
+	var/area/A = get_area(T)
+	var/message = "[ADMIN_LOOKUPFLW(user)]) has primed \a [name] for detonation at [ADMIN_COORDJMP(T)]"
```
Comment (Cyberboss) [inline code/game/objects/items/grenades/grenade.dm]: [src]

File: code/game/objects/items/grenades/grenade.dm [hunk 4]
```diff
@@ -50,26 +44,27 @@
...
+	GLOB.bombers += message
+	message_admins(message)
+	log_game("[key_name(user)] has primed \a [name] for detonation at [A.name] [COORD(T)].")
```
Comment (Cyberboss) [inline code/game/objects/items/grenades/grenade.dm]: [src]

File: code/game/objects/items/grenades/grenade.dm [hunk 5]
```diff
@@ -50,26 +44,27 @@
 	if(!active)
 		if(clown_check(user))
 			preprime(user)
-			if(iscarbon(user))
-				var/mob/living/carbon/C = user
-				C.throw_mode_on()
 
-/obj/item/grenade/proc/preprime(mob/user)
-	if(user)
+/obj/item/grenade/proc/glog(mob/user, turf/T)
```
Comment (Cyberboss) [inline code/game/objects/items/grenades/grenade.dm]: name this something more descriptive

**Review by Cyberboss — CHANGES_REQUESTED** [review]

**Review by Cyberboss — APPROVED** [review]

**Pull request merged**